README.md

```diff
@@ -127,6 +127,15 @@ see it, do the following:
 
 TODO(rachelrosen): Document and demo how to add convai-checker webcomponent outside of the angular-cli build using the imported compiled javascript.
 
+### Publishing the webcomponent
```

Comment:
Could you add a comment somewhere in here that this is for the plugin case?

Reply:
Done